index.js

```diff
@@ -109,6 +109,34 @@ class Executor {
         throw new Error('Not implemented');
     }
 
+    /**
+      * Starts a new frozen build in an executor
+      * @method _startFrozen
+      * @param {Object} config               Configuration
+      * @return {Promise}
+      */
+    startFrozen(config) {
+        return this._startFrozen(config);
+    }
+
+    async _startFrozen() {
+        throw new Error('Not implemented');
```

Comment:
Will need to add this function to `executor-k8s` and `executor-k8s-vm` to just resolves